themes/aframe/layout/partials/toc.ejs

```diff
@@ -0,0 +1,22 @@
+<% var toc = table_of_contents(page.content); %>
+
+<% if (toc.length) { %>
+  <h2>Table of Contents</h2>
+
+  <ul id="table-of-contents">
```

Comment:
I know `table-of-contents` is consistent with the other auto-generated IDs for headings. up to you.